core/src/main/java/com/vityuk/ginger/cache/AbstractLoadingCache.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * com.vityuk.ginger.cache.AbstractLoadingCache.java
+ *
+ * Copyright (c) 2007-2015 UShareSoft SAS, All rights reserved
```

Comment:
Just a reminder, this project is under Apache 2 license. Are you sure you have rights to release it under this license on behalf of UShareSoft? If yes, would you mind including full Apache 2 header in files where you do have a copyright. Thanks!